examples/demo_vxlan.pp

```diff
@@ -31,11 +31,21 @@
   }
 
   cisco_vxlan_vtep { 'nve1':
-    ensure             => present,
-    description        => 'Configured by Puppet',
-    host_reachability  => 'evpn',
-    shutdown           => 'false',
-    source_interface   => 'loopback55',
+    ensure            => present,
+    description       => 'Configured by puppet',
+    host_reachability => 'evpn',
+    shutdown          => 'false',
+    source_interface  => 'loopback55',
+  }
+
+  cisco_vxlan_vtep_vni {'nve1 10006':
+    ensure              => present,
+    assoc_vrf           => false,
+    ingress_replication => 'default',
```

Comment:
Let's add two examples in the demo manifest is possible showing both `ingress_replication` and `multicast_group`.  Maybe use different vni members for each?